environments/timelapse_env.yaml

```diff
@@ -0,0 +1,28 @@
+# env file
```

Comment:
What part of the timelapse is this environment for? Consider upgrading the comment to specify what this env is used for.

Reply:
added documentation